integration/container/inspect_test.go

```diff
@@ -64,3 +67,78 @@ func TestNetworkAliasesAreEmpty(t *testing.T) {
 		})
 	}
 }
+
+func TestInspectImageManifestPlatform(t *testing.T) {
+	skip.If(t, testEnv.IsRemoteDaemon)
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
+	skip.If(t, !testEnv.UsingSnapshotter())
+
+	tests := []struct {
+		name             string
+		image            string
+		skipIf           func() bool
+		expectedPlatform platforms.Platform
+	}{
+		{
+			name:  "amd64 only on any host",
+			image: "busybox:latest",
+			expectedPlatform: platforms.Platform{
+				OS:           "linux",
+				Architecture: "amd64",
+				Variant:      "",
+			},
+		},
+		{
+			skipIf: func() bool { return runtime.GOARCH != "amd64" },
+			name:   "amd64 image on non-amd64 host",
+
+			image: "hello-world:amd64",
+			expectedPlatform: platforms.Platform{
+				OS:           "linux",
+				Architecture: "amd64",
+			},
+		},
+		{
+			name:   "arm64 image on non-arm64 host",
+			skipIf: func() bool { return runtime.GOARCH != "arm64" },
+			image:  "hello-world:arm64",
+
+			expectedPlatform: platforms.Platform{
+				OS:           "linux",
+				Architecture: "arm64",
+				Variant:      "",
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		if tt.skipIf != nil && tt.skipIf() {
```

Comment:
Is it OK to use `tc` instead of `tt`? It's what we use in _most_ places (I know, I know... sorry!)

Reply:
Yes, good catch! I had the boilerplate generated by Copilot and didn't notice it used a different name 😅